Flake: TestDatastoreRequestHedging

```
--- FAIL: TestDatastoreRequestHedging (0.13s)
    --- FAIL: TestDatastoreRequestHedging/ReverseQueryRelationships (0.02s)
        hedging_test.go:158: PASS:	ReverseQueryRelationships(string,string)
        hedging_test.go:129: 
            	Error Trace:	/home/runner/work/spicedb/spicedb/internal/datastore/proxy/hedging_test.go:129
            	            				/home/runner/work/spicedb/spicedb/internal/datastore/proxy/hedging_test.go:[172](https://github.com/authzed/spicedb/actions/runs/3184619705/jobs/5193230142#step:4:174)
            	Error:      	Target error should be in err chain:
            	            	expected: "known error"
            	            	in chain: "another known error"
            	Test:       	TestDatastoreRequestHedging/ReverseQueryRelationships
```

--- a/internal/datastore/proxy/hedging_test.go
+++ b/internal/datastore/proxy/hedging_test.go
@@ -164,7 +164,7 @@ func TestDatastoreRequestHedging(t *testing.T) {
 
 			delegate.
 				On(tc.methodName, tc.arguments...).
-				WaitUntil(mockTime.After(3 * slowQueryTime)).
+				WaitUntil(mockTime.After(5 * slowQueryTime)).
 				Return(tc.firstCallResults...).
 				Once()
 			delegate.
@@ -172,7 +172,7 @@ func TestDatastoreRequestHedging(t *testing.T) {
 				Return(tc.secondCallResults...).
 				Once()
 
-			done := autoAdvance(mockTime, slowQueryTime, 5*slowQueryTime)
+			done := autoAdvance(mockTime, slowQueryTime, 6*slowQueryTime)
 
 			tc.f(t, proxy, false)
 			delegate.AssertExpectations(t)
@@ -181,16 +181,16 @@ func TestDatastoreRequestHedging(t *testing.T) {
 
 			delegate.
 				On(tc.methodName, tc.arguments...).
-				WaitUntil(mockTime.After(3 * slowQueryTime)).
+				WaitUntil(mockTime.After(7 * slowQueryTime)).
 				Return(tc.firstCallResults...).
 				Once()
 			delegate.
 				On(tc.methodName, tc.arguments...).
-				WaitUntil(mockTime.After(4 * slowQueryTime)).
+				WaitUntil(mockTime.After(8 * slowQueryTime)).
 				Return(tc.secondCallResults...).
 				Once()
 
-			autoAdvance(mockTime, slowQueryTime, 8*slowQueryTime)
+			autoAdvance(mockTime, slowQueryTime, 9*slowQueryTime)
 
 			tc.f(t, proxy, true)
 			delegate.AssertExpectations(t)

